### DISCUSSION TITLE
Metamask connection bug?

### DISCUSSION THREAD

**Discussion by orang3club** [subquery/subql#1065] [Frontier TestNet - Bug Reporting]
My indexer wallet is ***e401.
My node IP was 138.201.131.207
Once, when trying to enter the address 138.201.131.207:8000, and connected metamask, I saw information that I had to change the wallet ***e401 to the wallet ***69c0. This is also my wallet, from which I made a delegation to my indexer - ****e401. I didn't quite understand why I wasn't allowed to access using my indexer wallet and it was necessary to switch to a wallet, from which I just made a delegation and that's it. However, I was going to reinstall the node on another server anyway, so I just moved it and logged in already using the new IP.

**Comment by mzxyz** [comment]
If you admin app has registered or synced with account `***69c0` before , the service will keep these data.

Can you confirm that `***69c0` has been register as an indexer before , maybe unregistered, and you try to register `***e401` and then got this issue.

**Reply by orang3club** [reply]
No, I only registered one of my wallets as an indexer is ***e401. Maybe something happened by accident, but I definitely didn’t do it intentionally. And as far as it seems to me, I did not do anything to re-register the indexer wallet to another address.
